hadoop-ozone/dist/src/main/smoketest/s3/MultipartUpload.robot

```diff
@@ -148,6 +148,13 @@ Test Multipart Upload Complete
     ${result} =                 Execute AWSS3ApiCli        get-object --bucket ${BUCKET} --key ${PREFIX}/multipartKey1 --part-number 2 /tmp/${PREFIX}-multipartKey1-part2.result
     Compare files               /tmp/part2        /tmp/${PREFIX}-multipartKey1-part2.result
 
+#check whether parts count is retrieved through get and head object
+    ${result}                   Execute AWSS3ApiCli        get-object --bucket ${BUCKET} --key ${PREFIX}/multipartKey1 --part-number 1 /tmp/${PREFIX}-multipartKey1-part1.result
+                                Should contain             ${result}        PartsCount
```

Comment:
Check for the actual value of `PartsCount` in addition to just the presence of it.